dist/21-526EZ-ALLCLAIMS-schema.json

```diff
@@ -3322,14 +3157,68 @@
               "Medical Treatment Record - Government Facility",
               "Medical Treatment Record - Non-Government Facility",
               "DD214",
-              "Military Personnel Record",
-              "Decorations/Awards/Medal Citations",
               "Other Correspondence",
               "Buddy/Lay Statement"
             ]
           }
         }
       }
+    },
+    "unemployabilityAttachments": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "required": [
+          "name",
+          "attachmentId"
+        ],
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "confirmationCode": {
+            "type": "string"
+          },
+          "attachmentId": {
+            "type": "string",
+            "enum": [
+              "L149",
+              "L023"
+            ],
+            "enumNames": [
+              "VA 21-8940 Veterans Application for Increased Compensation Based on Unemployability",
+              "Other"
+            ]
+          }
+        }
+      }
+    },
+    "employmentRequestAttachments": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "required": [
+          "name",
+          "attachmentId"
+        ],
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "confirmationCode": {
+            "type": "string"
+          },
+          "attachmentId": {
+            "type": "string",
+            "enum": [
+              "L115"
```

Comment:
Could probably use [`const`](https://json-schema.org/latest/json-schema-validation.html#rfc.section.6.1.3) here instead of an enum of one.